app/api/leaderboard/functions.ts

```diff
@@ -32,17 +32,22 @@ export const getLeaderboardData = async (
   const shouldReverse = !ordering.startsWith("-");
 
   const contributors = await getContributors();
+  // console.log(contributors);
 
   const data = contributors
     .filter((a) => a.highlights.points)
     .map((contributor) => ({
       ...contributor,
       summary: contributor.summarize(...dateRange),
     }))
-    .filter((contributor) => contributor.summary.points)
+    .filter((contributor) => {
+      return contributor.summary.points;
+    })
     .sort((a, b) => {
       if (sortBy === "pr_stale") {
         return b.activityData.pr_stale - a.activityData.pr_stale;
+      } else if (sortBy == "new_contributors") {
+        return b.summary["points"] - a.summary["points"];
       }
```

Comment:
`new_contributors` is a boolean field, how would sort happen exactly? And why is it sorted by points here?

Sort by "date of first activity" instead